compiler/src/iree/compiler/API/Internal/IREEReduceToolEntryPoint.cpp

```diff
@@ -8,13 +8,13 @@
 #include "iree/compiler/tool_entry_points_api.h"
```

Comment:
What is the reason these changes are included as a part of this PR?

Reply:
There are some header include changes in other places caused this file filed to compile. So I fixed some header include issue. And along the way dropped the `llvm::` prefix which isn't necessary given `using namespace`.